iRacing/iRacing.cs

```diff
@@ -3834,7 +3788,7 @@ public override void DataUpdate()
 
                 double? behindGap = 0;
                 string behindName = "";
-                behindGlobal = behindName;
+                Globals.behindGlobal = behindName;
```

Comment:
Should also move aheadGlobal to Globals.cS?

Reply:
I think we should move the variables as we need them during refactoring. No need to move them to Globals in advance

Reply:
I see aheadGlobal is already moved to Globals.cs. Just that you're still using the "local" aheadGlobal in iRacing.cs.  Line 152 and 3777.

Reply:
oh... thx for the hint. Shall I change it?

Reply:
fixed

Reply:
... on the wrong branch ... now on correct one